source/deprecations/v2.x.html.md

```diff
@@ -876,6 +876,49 @@ let klass = factory.class;
 klass.hasSpeed('slow'); // true
 ```
 
+### Deprecations Added in 2.13
+
+#### EmberRouter.router renamed to EmberRouter.routerMicrolib
+
+##### until: 2.16.0
+##### id: ember-router.router
+
+The `router` property of `EmberRouter` has been renamed to `routerMicrolib` to disambiguate it from `router.js`, the microlib used within `EmberRouter`.
+
+Addon and application developers that are using the internal `router` property of `EmberRouter` should replace those usages with `EmberRouter.routerMicrolib`.
+
+This example demonstrates a common use case for `EmberRouter.router`. 
```

Comment:
Lets just say `.router` here (e.g. not `EmberRouter.router`, but just "the `.router` property).